### PR TITLE
log send prepare RPC for desktop

### PR DESCRIPTION
**Pull request by maxtaco**
@keybase/react-hackers  need this to only pay the price of logging on a  `log send`

### CODE REVIEW

File: go/client/cmd_log_send.go [hunk 1]
```diff
@@ -8,12 +8,12 @@ import (
...
 	"path/filepath"
 
-	"os"
-
 	"github.com/keybase/cli"
 	"github.com/keybase/client/go/install"
 	"github.com/keybase/client/go/libcmdline"
 	"github.com/keybase/client/go/libkb"
+	"golang.org/x/net/context"
+	"os"
```
Comment (patrickxb) [inline go/client/cmd_log_send.go]: can you move this up with other std packages?

**Comment by maxtaco** [comment]
How about if I fix the ordering when we fix our pre-commit hooks. There's a job for it in this sprint.

**Comment by patrickxb** [comment]
ok

**Comment by patrickxb** [comment]
@chrisnojima what happens if desktop app crashes?  all Keybase.app logs are lost?

**Comment by maxtaco** [comment]
@mmaxim says we should time this guy out if the electron UI is hosed

**Comment by patrickxb** [comment]
@chrisnojima do we also lose Keybase.app logs if user restarts the app?

**Comment by chrisnojima** [comment]
what will happen is electron will dump all actions in realtime but not dump out entire app state (very expensive). instead we dump that periodically and when we get this log send rpc we'll dump our ring buffer of states. So if electron dies or wahtever you won't have this larger set of dumps but with the actions and the period dumps it should still be useful

**Comment by maxtaco** [comment]
@mmaxim is this ok ^ ?

**Pull request merged**